cmd/flipt/main.go

```diff
@@ -368,6 +370,29 @@ func run(ctx context.Context, logger *zap.Logger, cfg *config.Config) error {
 	// starts REST http(s) server
 	g.Go(httpServer.Run)
 
+	if cfg.Server.Cloud.Enabled {
+		tunnel := fmt.Sprintf("%s-%s.%s", cfg.Server.Cloud.Instance, cfg.Server.Cloud.Organization, cfg.Server.Cloud.Address)
+
+		tunnelServer := &client.Server{
+			TunnelGroup:   tunnel,
```

Comment:
The `tunnel` address you have above is perfect for dialing, but for the `TunnelGroup` here, the new strategy is just `<org>.<cloud-address>:<port>`.